libs/datagrid/src/lib/cells/market-name-cell.tsx

```diff
@@ -1,39 +1,75 @@
-import type { MouseEvent, ReactNode } from 'react';
+import type { MouseEvent } from 'react';
 import { useCallback } from 'react';
 import get from 'lodash/get';
+import { Pill } from '@vegaprotocol/ui-toolkit';
+import type { Market } from '@vegaprotocol/types';
+
+const productTypeMap: { [key in string]: string } = {
```

Comment:
(non blocking) It would be better to use an `as const` pattern here for stricter types. Or better yet create a type using the schema?

```
const productTypeMap = {
  Future: 'Futr',
  Spot: 'Spot',
  Perpetual: 'Perp',
} as const;
type ProductType = keyof typeof productTypeMap
```

Reply:
changed to enum as const